workers/generic-worker/errorreport/errorreport.go

```diff
@@ -0,0 +1,28 @@
+package errorreport
+
+import (
+	"github.com/taskcluster/taskcluster/v30/internal/workerproto"
+)
+
+func Send(proto *workerproto.Protocol, err error, debugInfo map[string]string) {
+	title := "generic-worker error"
+	description := err.Error()
+	// could support differentiating for panics
+	kind := "error"
```

Comment:
maybe we call this `worker-error` or something to differentiate it from errors that come from w-m itself (in addition to how you've done it with the title)? Feel free to leave the same though.